controllers/seedimage_controller.go

```diff
@@ -539,6 +508,57 @@ func fillBuildImagePod(name, namespace, buildImg, baseImg, configMap string, pul
 	return pod
 }
 
+func defineInitContainers(baseImg, buildImg string, pullPolicy corev1.PullPolicy) []corev1.Container {
+	const baseIsoPath = "/iso/base.iso"
+
+	containers := []corev1.Container{}
+	buildCommands := []string{
+		fmt.Sprintf("xorriso -indev %s -outdev /iso/elemental.iso -map /overlay / -boot_image any replay", baseIsoPath),
+	}
+
+	if util.IsHTTP(baseImg) {
+		buildCommands = append([]string{fmt.Sprintf("curl -Lo %s %s", baseIsoPath, baseImg)}, buildCommands...)
+	} else {
+		// If baseImg is not an HTTP url assume it is an image reference
```

Comment:
Would it be worth having an image reference validation here? I am not sure, we have already other references in other places without any sort of validation. If we consider so I'd do it in a separate PR to try to validate any image reference provided by the user.

Reply:
well, don't know... It would be good to have validation of the built ISO, that's a good point.
What about doing a check after the ISO is built, and before publishing it?
Anyway, sure, I would not add anything in this PR. We may track it in a new issue.

Reply:
Not following you... There is no built done as this stage 😅 But yes, this sort of validations I'd move them to another issue as this is not exclusively related to this card.